langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -81,6 +81,11 @@ public const int UNSIGNED8_MAX_VALUE = 255;
 
 # Returns the absolute value of an int value.
 #
+# ```ballerina
+# int distance = -25;
```

Comment:
```suggestion
# int vectorDisplacement = -25;
```

Reply:
+1. or else we can use simple variable like `velocity`,if so we need to change `distance` variable accordingly.